server/config/config.go

```diff
@@ -563,7 +563,7 @@ type ScheduleConfig struct {
 	// SplitMergeInterval is the minimum interval time to permit merge after split.
 	SplitMergeInterval typeutil.Duration `toml:"split-merge-interval" json:"split-merge-interval"`
 	// SwitchWitnessInterval is the minimum interval that allows a peer to become a witness again after it is promoted to non-witness.
-	SwitchWitnessInterval typeutil.Duration `toml:"switch-witness-interval" json:"swtich-witness-interval"`
```

Comment:
Will it have a compatibility issue?

Reply:
This is a feature that has no GA and is only used within pd...
But I think it is possible to meet a compatibility issue

Reply:
not GA we can fix it directly.